src/screens/LoginPage/LoginPage.spec.tsx

```
@@ -2,9 +1,11 @@
+import React, { act, ChangeEvent, useEffect } from 'react';
```

ESLint

[error] 'ChangeEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)